modules/issue_tracker/php/edit.class.inc

```diff
@@ -0,0 +1,860 @@
+ *           Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class Edit extends \NDB_Page implements ETagCalculator
+{
+    /**
+     * This function will return a json response.
+     *
+     * @param ServerRequestInterface $request The incoming PSR7 request
+     *
+     * @return ResponseInterface The outgoing PSR7 response
+     */
+    public function handle(ServerRequestInterface $request) : ResponseInterface
+    {
+        // Ensure GET or POST request.
+        switch ($request->getMethod()) {
+        case 'GET':
+            return $this->_handleGET($request);
+        case 'POST':
+            return $this->_handlePOST($request);
+        default:
+            return new \LORIS\Http\Response\JSON\MethodNotAllowed(
+                $this->allowedMethods()
+            );
+        }
+    }
+
+    /**
+     * _handleGET retrieve the edit issue details
+     *
+     * @param ServerRequestInterface $request The incoming PSR7 request.
+     *
+     * @return ResponseInterface The outgoing PSR7 response
+     */
+    private function _handleGET(ServerRequestInterface $request) : ResponseInterface
+    {
+        // Parse GET query params.
+        $values = $request->getQueryParams();
+
+        $user = $request->getAttribute('user');
+        $db   = $this->loris->getDatabaseConnection();
+
+        // get field options
+        $sites = Issue_Tracker::getSites(false, true);
+
+        //not yet ideal permissions
+        $assignees      = [];
+        $inactive_users = [];
+        if ($user->hasPermission('access_all_profiles')) {
+            $assignee_expanded = $db->pselect(
+                "SELECT Real_name, UserID FROM users",
+                []
+            );
+
+            $inactive_users_expanded = $db->pselect(
+                "SELECT Real_name, UserID FROM users
+                  WHERE Active='N'",
+                []
+            );
+        } else {
+            $CenterID = implode(',', $user->getCenterIDs());
+            $DCCID    = $db->pselectOne(
+                "SELECT CenterID from psc where Name='DCC'",
+                []
+            );
+            $assignee_expanded = $db->pselect(
+                "SELECT DISTINCT u.Real_name, u.UserID FROM users u
+                 LEFT JOIN user_psc_rel upr ON (upr.UserID=u.ID)
+                 WHERE FIND_IN_SET(upr.CenterID,:CenterID) OR (upr.CenterID=:DCC)",
+                [
+                    'CenterID' => $CenterID,
+                    'DCC'      => $DCCID,
+                ]
+            );
+
+            $inactive_users_expanded = $db->pselect(
+                "SELECT DISTINCT u.Real_name, u.UserID FROM users u
+                 LEFT JOIN user_psc_rel upr ON (upr.UserID=u.ID)
+                 WHERE FIND_IN_SET(upr.CenterID,:CenterID) OR (upr.CenterID=:DCC)
+                 AND Active='N'",
+                []
```

Comment:
This doesn't look like a valid query? There is nothing bound to the bind parameters.

Reply:
@driusan wow I'm something went wrong awhile back in the code because it's like that on main.